Gemfile

```diff
@@ -74,4 +74,5 @@ group :test do
   gem 'sqlite3', platforms: :mri
   gem 'activerecord-jdbcsqlite3-adapter', platforms: :jruby
   gem 'poltergeist'
+  gem 'tins', '1.6.0'
```

Comment:
Will do. 

What does that statement do on Ruby 2.*? 

Is it effectively ignored in Rubies later than 1.9 so that later versions of the _tins_ gem (`1.11.0`) are installed?

Committing now. 




> On Aug 3, 2016, at 3:36 PM, Igor Fedoronchuk <notifications@github.com> wrote:
> 
> In Gemfile <https://github.com/activeadmin/activeadmin/pull/4551#discussion_r73405426>:
> 
> > @@ -74,4 +74,5 @@ group :test do
> >    gem 'sqlite3', platforms: :mri
> >    gem 'activerecord-jdbcsqlite3-adapter', platforms: :jruby
> >    gem 'poltergeist'
> > +  gem 'tins', '1.6.0'
> @martinstreicher <https://github.com/martinstreicher> , try
> 
> gem 'tins', '< 1.3.4', platforms: :ruby_19
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/activeadmin/activeadmin/pull/4551/files/400cc97f161b71876965f57b307d0a0f54392990#r73405426>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAB6VwPLa3t7bNcpJs7zvmq3WZUKuH6qks5qcO3YgaJpZM4JYflM>.
>